src/app/dataScripts/fetchSpendingOverTimeData.ts

```diff
@@ -0,0 +1,102 @@
+import fs, { mkdir } from 'node:fs/promises';
+import { existsSync } from 'node:fs';
+
+import path from 'node:path';
+
+import { spendingApiUrl } from '../src/constants';
+import { getSpendingOverTimeByStateRequest } from '../src/util';
+import {
+    MonthlySpendingOverTime,
+    MonthlySpendingOverTimeByState,
+    MonthlySpendingOverTimeResponse,
+} from '../src/types/api';
+
+import { dataDir } from './nodeConstants';
+import { STATE_CODES } from './stateCodes';
```

Comment:
We're already fetching state data in fetchStatesData.ts. Perhaps we can get the state codes from there instead?

Reply:
Nice, yup! That fix is in 1eafe2eb098bb2596d943ee0dab784b5fe3f0b2d